sotodlib/core/context.py

```diff
@@ -329,6 +340,87 @@ def get_obs(self,
                         _det_info.move(k, None)
                 meta.det_info.merge(_det_info)
             aman.merge(meta)
+
+        # Deal with special channels, if they exist.
+        # We will merge the dets and tdets axes
+        # And merge the tdets signal into the dets signal, band, and channels.
+        # nans will be inserted into every other dataset assigned to the dets axis.
+        if special_channels and reindex_dets:
```

Comment:
Please add `and 'tones' in aman`, reindex_dets will crash when tones do not exist.
'det_info.smurf' is also required for reindex_dets.